inc/class-core-sitemaps-index.php

```diff
@@ -86,6 +83,7 @@ public function render_sitemap() {
 	 *
 	 * @param string $output robots.txt output.
 	 * @param bool   $public Whether the site is public or not.
+	 *
```

Comment:
Remove the extra line here. Not sure if this is an automatic thing you can change in your editor config or not, but would be good to see.